src/components/Blocking/SpecificIssue.js

```diff
@@ -90,12 +99,15 @@ const styles = StyleSheet.create({
     paddingRight: 13,
     paddingLeft: 19,
     fontFamily: fonts.inter[400],
-    color: COLORS.blackgrey
+    color: COLORS.white2,
+    borderWidth: 1,
+    borderColor: COLORS.gray200
   },
   container: {
     height: 'auto',
     borderTopRightRadius: 20,
-    borderTopLeftRadius: 20
+    borderTopLeftRadius: 20,
+    backgroundColor: COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of a background color to the container is a good practice for ensuring consistent UI across different devices and screen settings. The chosen color `COLORS.almostBlack` should be fine as long as it aligns with the overall design theme and doesn't affect readability or accessibility.


<!-- This is an auto-generated comment by OSS CodeRabbit -->